content/en/logs/indexes/_index.md

```diff
@@ -78,6 +78,37 @@ Update or remove this quota at any time when editing the Index:
 
 **Note**: Indexes daily quotas reset automatically at 2:00pm UTC (4:00pm CET, 10:00am EDT, 7:00am PDT).
 
+### Attribute based sampling
+
+In order to ensure complete log information related to a given subset of transactions, sample on a specific attribute holding an ID, such as User, Session, Request Identifiers.
+By default, the log-based sampling is applied (exclusion percentage set on `all logs`).
+
+This feature operates on attribute's values rather than the log itself, for example:
+* given an attribute `id` holding 3 distinct values (`1`, `2`, `3`),
+* defining a 66% exclusion filter on it,
+* results in indexing all logs holding 1 specific attribute's value (`3`), all the other logs won't be indexed.
+
+{{< img src="logs/indexes/index_exclusion_attribute_groups.png" alt="index attribute-based exclusion filter groups"  style="width:70%;">}}
+
+For example, exclude `nginx` logs for 99% of `http.request_id`:
+
+{{< img src="logs/indexes/index_exclusion_on_attribute.png" alt="index attribute-based exclusion filter"  style="width:70%;">}}
+
+**Note**: 
+
+* You must prefix the attribute name with the `@` character.
+
+* For low cardinality attributes, you should define a set of distinct Index Exclusion Filters on `all logs`.
+
+#### Sample all logs of a trace
+
+In order to guarantee the sampling of all logs related to a given trace, define the log sampling on `Trace Id` attribute.
```

Comment:
We could be clearer here, as I'm not sure it's crystal clear what we are trying to say.

Something like the following:


Once [logs and traces are connected], a `trace_id` attribute is automatically added in the logs. In order to be able to sample those logs but make sure that for the chosen trace ids the full set of logs is kept, an attribute-based exclusion filter should be set on the `trace_id` attribute.

For example, setting exclude 60% of `Trace Id`, results in:

* 40% of all traces get all their related logs indexed.
* 60% of all traces have their logs sampled (still available in livetail and archives)

Reply:
Actually the last bullet is not correct: for the 60% of traces we don't get any logs indexed.